browser.js

```diff
@@ -292,16 +274,15 @@ function zoomInit() {
 }
 
 document.addEventListener('DOMContentLoaded', () => {
-	// TODO: Figure out how to loud external scripts here
-	// [
-	// 	path.resolve('vendor/mousetrap.js'),
-	// 	path.resolve('vendor/mousetrap-global-bind.js')
-	// ].forEach(src => {
-	// 	const script = document.createElement('script');
-	// 	script.src = `file://${src}`;
-	// 	script.async = false;
-	// 	document.head.appendChild(script);
-	// });
```

Comment:
Chrome doesn't allow webpages to load inline scripts in that way so they were just blocked.